src/scripts/librarymenu.js

```diff
@@ -724,7 +724,8 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
     }
 
     function initHeadRoom(elem) {
-        require(["headroom-window"], function (headroom) {
+        require(["headroom"], function (Headroom) {
+            var headroom = new Headroom([], {});
```

Comment:
why change?

Reply:
because headroom-window is headroom! why is that? because someone remembered of defining in site.js a module named headroom-window that loads headroom! xD the wonderful things one can discover in this project..